telink: Update AndeSight Toolchain

s07641069 opened the pull request:
Update from AndeSight v5.3.3 Toolchain (GCC 12.2) to AndeSight V5.4.1 Toolchain (GCC 14.2)

andriy-bilynskyy commented:
The following west manifest projects have been modified in this Pull Request:

| Name | Old Revision | New Revision | Diff |
| ---- | ------------ | ------------ |------|
| hal_telink | https://github.com/telink-semi/hal_telink/commit/1b49a548eba87a4e3c617de027e8c2a41a25d69b | https://github.com/telink-semi/hal_telink/commit/0c2d784cb4d49a04878a22f5dda589605dd0d4c4 (develop) | [telink-semi/hal_telink@1b49a548..0c2d784c](https://github.com/telink-semi/hal_telink/compare/1b49a548eba87a4e3c617de027e8c2a41a25d69b..0c2d784cb4d49a04878a22f5dda589605dd0d4c4) |

*Note: This message is automatically posted and updated by the Manifest GitHub Action.*

s07641069 committed:
telink: Update AndeSight Toolchain

- Update AndeSight v5.3.3 (GCC 12.2) to AndeSight V5.4.0 (GCC 14.2)
- Аix include-fixed path handling for GCC 13+

Signed-off-by: Alex Tsitsiura <s07641069@gmail.com>
